THIRD_PARTY_LICENSES.txt

```diff
@@ -1,114 +1,114 @@
- pytz                      2022.6     MIT License                                                                                         
- readme-renderer           37.3       Apache Software License                                                                             
- requests                  2.28.1     Apache Software License                                                                             
- requests-toolbelt         0.10.1     Apache Software License                                                                             
- rfc3986                   2.0.0      Apache Software License                                                                             
- rich                      12.6.0     MIT License                                                                                         
- six                       1.16.0     MIT License                                                                                         
- sqlparse                  0.4.3      BSD License                                                                                         
- stack-data                0.6.0      MIT License                                                                                         
- text-unidecode            1.3        Artistic License; GNU General Public License (GPL); GNU General Public License v2 or later (GPLv2+) 
- toml                      0.10.2     MIT License                                                                                         
- tomli                     2.0.1      MIT License                                                                                         
- tox                       3.27.0     MIT License                                                                                         
- traitlets                 5.5.0      BSD License                                                                                         
- twine                     4.0.1      Apache Software License                                                                             
- typed-ast                 1.4.3      Apache License 2.0                                                                                  
- typing_extensions         4.4.0      Python Software Foundation License                                                                  
- urllib3                   1.26.12    MIT License                                                                                         
- virtualenv                20.16.6    MIT License                                                                                         
- webencodings              0.5.1      BSD License                                                                                         
- zipp                      3.10.0     MIT License
\ No newline at end of file
+Name                      Version    License
```

Comment:
Should we add a line about how this is generated?

Reply:
I generated it by having an empty workspace, installing dbt-dremio and running `pip-licenses`

https://pypi.org/project/pip-licenses/

Reply:
Added a comment at the beginning of the file.